web/modules/custom/rdf_entity/src/Entity/Query/Sparql/Query.php

```diff
@@ -154,7 +162,12 @@ public function condition($property, $value = NULL, $operator = NULL, $langcode
       case 'label-=':
         preg_match('/\((.*?)\)/', $value, $matches);
         $matching = array_pop($matches);
-        $ids = "(<$matching>)";
+        if($matching){
+          $ids = "(<$matching>)";
+        }
+        else{
+          $ids = '("' . $value . '")';
+        }
```

Comment:
Coding standards are not correct here. Did the bot accept this? Then we should look into why this file is not checked for coding standards.